pkg/theme/theme.go

```diff
@@ -0,0 +1,68 @@
+package theme
```

Comment:
seems like there is some overlap between this file and `pkg/gui/theme.go`. Perhaps we could combine the two files?

Reply:
Fixed